ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,244 @@
+"""
+The RQ integration will trace your jobs.
+
+Usage
+~~~~~
+
+The rq integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(rq=True)
+
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_RQ_WORKER_SERVICE=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq
+for a working example.
+
+
+Instance Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.rq['distributed_tracing_enabled']
+
+   If ``True`` the integration will connect the traces sent between the queuer
+   and the RQ worker.
+
+   Default: ``False``
+
+.. py:data:: ddtrace.config.rq['service']
+
+   The service name reported by default for RQ spans from the app.
+
+   This option can also be set with the ``DD_RQ_SERVICE`` or ``DD_SERVICE``
+   environment variables.
+
+   Default: ``rq``
+
+.. py:data:: ddtrace.config.rq_worker['service']
+
+   The service name reported by default for RQ spans from workers.
+
+   This option can also be set with the ``DD_RQ_WORKER_SERVICE`` environment
+   variable.
+
+   Default: ``rq-worker``
+"""
+from ddtrace import Pin
+from ddtrace import config
+
+from .. import trace_utils
+from ...ext import SpanTypes
+from ...propagation.http import HTTPPropagator
+
+
+__all__ = [
+    "patch",
+    "unpatch",
+]
+
+
+config._add(
+    "rq",
+    dict(
+        _default_service="rq",
+        distributed_tracing_enabled=False,
+    ),
+)
+
+config._add(
+    "rq_worker",
+    dict(
+        _default_service="rq-worker",
+    ),
+)
+
+
+propagator = HTTPPropagator()
```

Comment:
I think all the methods of `HTTPPropagator` are static so no need of an instance

Reply:
Yup this has changed since this PR was started 😆